package.json

```diff
@@ -37,6 +37,7 @@
     "eslint": "^8.37.0",
     "ini": "^4.0.0",
     "luxon": "^3.2.1",
+    "material-design-icons": "^3.0.1",
```

Comment:
Looks like this package is 7 years old? https://www.npmjs.com/package/material-design-icons?activeTab=versions

That's the only official NPM package though, so maybe it's ok for now?